contrib/kube-prometheus/manifests/grafana/grafana-dashboards.yaml

```diff
@@ -7495,4 +7495,3 @@ data:
         "type": "prometheus",
         "url": "http://prometheus-k8s.monitoring.svc:9090"
     }
----
```

Comment:
I am aware this file is generated by `contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/bin/grafana_dashboards_generate.sh`, manually editing is not recommended. I will fix that asap.